src/cryptonote_core/service_node_rules.cpp

```diff
@@ -21,7 +21,7 @@ uint64_t get_staking_requirement(cryptonote::network_type m_nettype, uint64_t he
   // (proportionally) whatever staking changes happen on mainnet.  (The 650k is because devnet
   // launched at ~600k mainnet height, so this puts it a little ahead).
   if (m_nettype == cryptonote::DEVNET)
-      return get_staking_requirement(cryptonote::MAINNET, 600000 + height, hf_version) / 10;
+      return COIN * 100;
```

Comment:
Should just delete all the above, plus comment, and just add a ` || m_nettype == cryptonote::DEVNET` into the earlier if.